backend/frontend/views.py

```diff
@@ -50,22 +57,43 @@
 ]
 
 
-from django.contrib.auth.decorators import user_passes_test
-
-
 def is_super(user):
     return user.is_superuser
 
 
-@user_passes_test(is_super)
+def has_view_access(logged_user):
```

Comment:
logged_user? You mean logged_in_user?

Reply:
Also, sometimes a person can have view access over a whole team. This function name can be more explicit.

Maybe `can_view_user` or something like that? Then we can do another one for `can_view_team`. Or is the intention of this function to be used on user views and team views?

Reply:
I did originally put the is_super function in views.py, but it was because it is a 2 line function we will probably throw away. This is getting a bit long.

Can you think of a better way to organize the code? 

I also think it would be good to write tests.